<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="1" failed="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="26" started-at="2016-07-28T09:23:57Z" finished-at="2016-07-28T09:23:57Z">
    <groups>
    </groups>
    <test name="Regression suite" duration-ms="26" started-at="2016-07-28T09:23:57Z" finished-at="2016-07-28T09:23:57Z">
      <class name="test.java.scripts.TestLoginAddBeneficiary">
        <test-method status="FAIL" signature="beforeClass()[pri:0, instance:test.java.scripts.TestLoginAddBeneficiary@47c62251]" name="beforeClass" is-config="true" duration-ms="0" started-at="2016-07-28T14:53:57Z" finished-at="2016-07-28T14:53:57Z">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at test.java.Utils.SeleniumDriver.beforeClass(SeleniumDriver.java:28)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:510)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:211)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:138)
	at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:170)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:104)
	at org.testng.TestRunner.privateRun(TestRunner.java:773)
	at org.testng.TestRunner.run(TestRunner.java:623)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1185)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1110)
	at org.testng.TestNG.run(TestNG.java:1018)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:152)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:57)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeClass -->
        <test-method status="SKIP" signature="execute()[pri:0, instance:test.java.scripts.TestLoginAddBeneficiary@47c62251]" name="execute" duration-ms="1" started-at="2016-07-28T14:53:57Z" finished-at="2016-07-28T14:53:57Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- execute -->
        <test-method status="SKIP" signature="afterMethod(org.testng.ITestResult)[pri:0, instance:test.java.scripts.TestLoginAddBeneficiary@47c62251]" name="afterMethod" is-config="true" duration-ms="0" started-at="2016-07-28T14:53:57Z" finished-at="2016-07-28T14:53:57Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method status="SKIP" signature="afterClass()[pri:0, instance:test.java.scripts.TestLoginAddBeneficiary@47c62251]" name="afterClass" is-config="true" duration-ms="0" started-at="2016-07-28T14:53:57Z" finished-at="2016-07-28T14:53:57Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterClass -->
      </class> <!-- test.java.scripts.TestLoginAddBeneficiary -->
    </test> <!-- Regression suite -->
  </suite> <!-- Suite -->
</testng-results>
